Finished, Quest02 CSS Quest

RohMoon opened the pull request:
# PR 
WebDevCurriculum Quest
##  Content
 - modified    
   - Quest01 HTML CODE Structure Changed.
   - Quest02 CSS Code Changed.

RohMoon committed:
addited abo blahblah

RohMoon committed:
내용 변경후 Stash 사용

RohMoon committed:
체크리스트 내용  두줄 추가

RohMoon committed:
형상관리 4구조 추가

RohMoon committed:
형상관리 4가지 단순설명 추가

RohMoon committed:
추가 내용 명령어 절반

RohMoon committed:
체크리스트 명령어 파트 완료

RohMoon committed:
' 깃 히스토리 생성 방식 ' 제외 모든 checkList 에 대한 답변 작성

RohMoon committed:
내부구조 추가 완료

RohMoon committed:
Merge pull request #1 from RohMoon/NextLvl_00

Quest 00 체크리스트 example 파일에 추가 Rebase cherry Pick등 명령어 연습 훨씬 많은량 요구

RohMoon committed:
Advanced 추가

RohMoon committed:
hTMl 역사 메모 추가

RohMoon committed:
"오탈자 확인 및 md포맷으로 변경 "

RohMoon committed:
Merge pull request #2 from RohMoon/NextLvl_00

오탈자 확인 및 md포맷으로 변경

RohMoon committed:
HTML 재작성 중간 저장

RohMoon committed:
XML 간략 설명까지 완료

RohMoon committed:
점유율 이전까지 작성

RohMoon committed:
모바일 브라우저의 특징 이전까지 작성

RohMoon committed:
완성

RohMoon committed:
Quest00 .md파일 변경

RohMoon committed:
test

RohMoon committed:
Quest00 .md파일 수정

RohMoon committed:
Quest01 HTML
* element 생성 및 1차 class 분배

RohMoon committed:
Quest01 HTML
* 체크

RohMoon committed:
Quest02 . CSS Quest 및 checkList 추가

RohMoon committed:
Quest02 . CSS Quest 및 checkList 추가

RohMoon committed:
Quest02 . CSS Quest 및 checkList 추가

RohMoon committed:
Quest02 . greegot, w_new .css 파일
원 m.naver.com에서 발췌하여 붙여봄

RohMoon committed:
git commit #force

RohMoon committed:
Merge branch 'master' of https://github.com/RohMoon/WebDevCurriculum

RohMoon committed:
Quest02 . Add nar's index

RohMoon committed:
Merge remote-tracking branch 'origin/master'

RohMoon committed:
Deleted CheckList,greedot.css, w_new.css

RohMoon committed:
Merge pull request #3 from RohMoon/20211209

modified : DeletedFile

RohMoon committed:
Modified Quest02
 -index.html : Loge And nav.

RohMoon committed:
Modified Quest02
 -index.html 구조 변경
 -CSS 재정리
 ** 날씨 정보까지 **

RohMoon committed:
Modified Quest02
 -index.html 구조 변경
 -CSS 재정리
 ** 날씨 정보까지 **

RohMoon committed:
Modified Quest02
 - index.html
 * 완료 최종본

RohMoon commented:
- Status : Ready for Sign to Merge.

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
-			<!-- TODO: 하단의 여러 가지 링크들은 어떻게 처리하면 좋을까요? -->
-		</footer>
-	</body>
+<html xmlns="http://www.w3.org/1999/html">
```
xmlns 를 사용한 이유는 ??

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
+    <meta charset="UTF-8">
+    <script src="https://use.fontawesome.com/releases/v5.2.0/js/all.js"></script>
+    <link rel="stylesheet" href="../../Quest02/q02_css/main.css">
+    <link rel="stylesheet" href="../../Quest02/q02_css/reset.css">
```
여러 stylesheet를 선언한다면 어느것이 가장 마지막에 적용될까 ?

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
+        <div class="searchBar">
+            <input
+                    type="text"
+                    class="SearchBar_Input"
```
PascalCase / camelCase / snake_case 등을 혼재해서 사용하는건 좋은방식이 아님

html 내에서 사용된 다른 class name들과 동일한 포맷을 유지할 것

nunch-dev replied:
https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
+                        <img
+                                class="list-img"
+                                src="./img/icons/icon_shop.png"
+                        />
```
img 태그 내의 alt attr은 무슨 역할을 할까 ?

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
+                    </a>
+                    <p class="list-content">카페</p>
+                </li>
+                </li>
```
태그의 열고 닫힘은 확실하게

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
+                <button id="weekSomkeInfo" class="weatherSmokeInfoButtonSquare">주간 미세먼지</button>
+            </div>
+            <div id="weatherSmokeInfo_displayPanel">
+                <img src="./img/weather/weeklyweather.png"/>
```
이미지~

nunch-dev commented on Quest01/skeleton/index.html:
```diff
@@ -1,20 +1,315 @@
+
+        <div class="recommendOurServiceApp A_GradeContainer">
+            <div class="recommendOurServiceApp-img">
+                <img src="./img/icons/sp_search_9655b415.png"/>
```
스프라이트 이미지 사용방법 확인

nunch-dev commented on Quest02/from_narin_css/reset.css:
```diff
@@ -0,0 +1,13 @@
+    width: 100%;
+}
+:root {
+    --main-color: #19ce60;
```
이건 무슨 용도로 사용한걸까?

nunch-dev commented on Quest02/q02_css/main.css:
```diff
@@ -0,0 +1,449 @@
+}
+.SearchBar_Input{
+    padding-left: 0.5rem;
+    width: inherit;
+    height: inherit;
```
inherit를 사용한 이유는?

nunch-dev commented on Quest02/q02_css/main.css:
```diff
@@ -0,0 +1,449 @@
+    text-align: center;
+    font-size: 2.5em;
+    font-weight: bolder;
+    font-family: Impact;
```
Impact는 무엇 ?

nunch-dev commented on Quest02/q02_css/main.css:
```diff
@@ -0,0 +1,449 @@
+    box-sizing : border-box; /* 테두리와 안쪽 여백의 크기도 요소의 크기로 고려한다. 너비를 100픽셀로 설정하고 테두리와 안쪽 여백을 추가하면, 콘텐츠 영역이 줄어들어 총 너비 100픽셀으 유지한다. 대부분의 경우 이 편이 크기를 조절할 때 쉽다.*/
+                             /*content-box로 지정하면 기본 css 박스 크기 결정법을 사용한다. 요소의 너비를 100 픽셀로 설정하면 콘텐츠 영역 100픽셀 너비를 가지고, 테두리와 안쪽 여백은 이에 더해진다.*/
+    width:24%;
+    margin-right: calc(4%/3);
```
`calc` 는 무슨 역할을 할까?

nunch-dev commented on Quest02/q02_css/main.css:
```diff
@@ -0,0 +1,449 @@
+}
+header #titleLogo{
+    text-align: center;
+    font-size: 2.5em;
```
`em`과 `rem`의 차이는 ?

nunch-dev commented on Quest02/q02_css/main.css:
```diff
@@ -0,0 +1,449 @@
+}
+.list-img{}
+.list-content{
+    font-siez : 0.8rem
```
오타

nunch-dev commented on Quest02/q02_css/main.css:
```diff
@@ -0,0 +1,449 @@
+    position: absolute;
+    width: 465px;
+    height: 400px;
+    clip: rect(0px,66px,66px,0px);
```
clip property는 무슨역할을 하나 ?

RohMoon committed:
Modified Quest02
 - 수정사항
  - alias 명칭 방식 통일
  - 일부 그림 background방식 이용
  - 주간 그림 html 구조로 변경
  - css 추가 및 수정
